roles/setup_efm/tasks/efm_configure.yml

```diff
@@ -37,6 +37,12 @@
 - name: Prepare final parameters list
   set_fact:
     efm_node_parameters: "{{ efm_default_parameters + efm_node_parameters }}"
+  when: item.value.upstream_node is not defined
```

Comment:
Please use `upstream_node_private_ip`. We use this key to sort the server list: https://github.com/EnterpriseDB/edb-ansible/blob/REL-2_0_3/filter_plugins/sort_servers.py#L90

Reply:
Fixed